modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointment.rb

```diff
@@ -110,6 +110,10 @@ def build_appointment_model
           }
 
           StatsD.increment('mobile.appointments.type', tags: ["type:#{appointment_type}"])
+          # i believe these two were to help us determine which values were possible. it copies the older version of this file.
+          # it's plausible there could still be values we don't know about, but we also aren't actually checking these logs
+          # so it's useless to us anyhow. what would potentially be more valuable would be outputting when they're not in the
+          # known list. we should definitely remove these
           Rails.logger.info('metric.mobile.appointment.type', type: appointment_type)
           Rails.logger.info('metric.mobile.appointment.upstream_status', status: appointment[:status])
```

Comment:
Category: data discovery
Action: remove. this has been around a while and we're not actively monitoring it